apps/site/lib/site_web/views/trip_plan_view.ex

```diff
@@ -473,17 +475,33 @@ defmodule SiteWeb.TripPlanView do
 
       itinerary_is_from_or_to_airport = itinerary_satisfies_property?(i, :is_from_or_to_airport)
 
+      itinerary_contains_blue_line = itinerary_satisfies_property?(i, :contains_blue_line)
+
+      itinerary_contains_east_boston_ferry =
+        itinerary_satisfies_property?(i, :contains_east_boston_ferry)
+
+      itinerary_contains_newburyport_rockport_line =
+        itinerary_satisfies_property?(i, :contains_newburyport_rockport_line)
+
       show_fares =
         !itinerary_satisfies_property?(i, :contains_capeflyer) || one_way_total_fare != 0
 
+      is_sumner_tunnel_closed_period =
+        Date.compare(date, ~D[2023-07-05]) != :lt && Date.compare(date, ~D[2023-08-31]) != :gt
```

Comment:
**issue**: `date` will be the date you're using the website, not necessarily the date being planned for (the user can pick a different date in the date picker). I'd recommend using the itinerary `start` date.